### PR TITLE
Do not show inserter while left mouse button is pressed

### PR DESCRIPTION
**Pull request by flyingbee2012**
A behavior in parity with Word

### CODE REVIEW

File: packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts [hunk 1]
```diff
@@ -73,13 +74,18 @@ export default class TableResize implements EditorPlugin {
         this.editor = editor;
         this.setupResizerContainer();
         this.onMouseMoveDisposer = this.editor.addDomEventHandler('mousemove', this.onMouseMove);
+        this.editor.addDomEventHandler('mousedown', this.onMouseDown);
+        this.editor.addDomEventHandler('mouseup', this.onMouseUp);
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:78]: no need to do this, you can directly get button info from MouseEvent.which from mousemove event

Comment (flyingbee2012) [reply packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:78]: Good to know that, I was trying to use e.button the MouseEvent inside onMouseMove but it didn't work.

Comment (flyingbee2012) [reply packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts]: Used e.buttons instead as e.which seems deprecated.

File: packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts [hunk 2]
```diff
@@ -194,11 +194,14 @@ export default class TableResize implements EditorPlugin {
                                 }
                                 if (verticalInserterTd) {
                                     this.setCurrentTd(null);
-                                    this.setCurrentInsertTd(
-                                        ResizeState.Vertical,
-                                        verticalInserterTd,
-                                        map.rect
-                                    );
+                                    // we hide the inserter if left mouse button is pressed
+                                    if (e.buttons !== 1) {
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:198]: what if I press both left and right button? Will buttons be 2? I prefer check buttons == 0 here

File: packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts [hunk 3]
```diff
@@ -194,11 +194,14 @@ export default class TableResize implements EditorPlugin {
                                 }
                                 if (verticalInserterTd) {
                                     this.setCurrentTd(null);
-                                    this.setCurrentInsertTd(
-                                        ResizeState.Vertical,
-                                        verticalInserterTd,
-                                        map.rect
-                                    );
+                                    // we hide the inserter if left mouse button is pressed
+                                    if (e.buttons == 0) {
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:198]: Actually, why not just do this check at very beginning of this function (line 130), and return directly if buttons is not 0? Not need to do all further calculation.

Comment (JiuqingSong) [reply packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts:198]: Oh I see, so in the dragging scenario, onmousemove still need to work.

**Review by JiuqingSong — APPROVED** [review]

**Pull request merged**